src/code/models/node.ts

```diff
@@ -144,8 +147,8 @@ export class Node extends GraphPrimitive {
-          || val === "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACgAAAAoCAYAAACM/rhtAAAAAXNSR0IArs4c6QAABtlJREFUWEftmG1oW1UYx8/JfUtf0pu3bs2abMtcVremXxz74IQNter0w0DUgoNVxUF8q3X5Nj+IoJuILtGxYRBFpuAHYX5xgsKkMlYEt87ibEbWUJaZptnyni5p0t57j/zveof4sqbZxAkeuOTem3PP+d3/83+ecxJKbvNGm+FjjOE5jhBiisfjlOf5FlVVxUqlog/X1tZGJElSFEWpKorC6vU66+3tXaCUsuXO1zRgoVDwF4vFrmq1ahJFca+qqveWy2VR0zTS0dFBeJ4fr1Qqb9tstoIkSdV6vX7a6/XW/lHAdDrddvXq1cey2ezuSqVyN8RaWFggqqoCSD/QjHsmk+n6PYvFct5qtX4py/L7Lpcr0yhowwoyxsyxWOytQqHwVLFYtLW0tBCLxULy+TzJ5XLE4XAQl8ulA12+fJmk02kdtL29nVBK9cNmsxGn03nSZrM90dXVdaURyIYBAXXp0qXjqVRqKyZGGCVJIqVSSQc0FJRlmczMzBD40ePxkDVr1ujfFYtFvV93d3fO6XTetWrVql8b8WTDgGfOnBHsdvvT+Xx+qFKp3KlpmiCKIoHnstmsDguVENa5uTlSrVbJihUrdAXr9boulizLeYvFsn9ubu6w3++fv6UKGoOlUqk76/X6K9Vqdefs7KyjXC4LiqJQgKHBjzgHOA5ZlhcsFosiy/KPPM8f8Hg831FK1Ubg0KdhBX8/4GKZscdisWej0ehwPp9fJQiC3kVRFF1J2ACq9vb2fub1eg84HI7J5YAZ8y0JyBgTstnsPaqqWhhjmziO+wElw+PxzCUSiYFUKnVgcnLyjkKhQBhjuoIARC1ct24dPPemIAjv+Hy+MqqAJEkPVKtVVRTFzRzHhW02W/lGXlwSMBqNPpLL5Q4riuI1m81EFMVfNE0b3LJly0+MMVM6nT4QjUZfzmQyLUZoAdna2ko2bdo0IoriLq/Xm4YiExMTh4rF4nPwL5Tu7OwcN5vNO9evX//r34V8ScDTp09Hzp8/H8AAMLvdbp/u6Oh4ur+///tMJtMXj8ffnZiY2M7zPIcMBuTs7CyBomvXrr3i8XheslqtX6FIj4yMfDU1NfVwe3s7h8Ty+/1EEATf1q1b400Dnjt3LnLq1KmA0+kk5XIZCqJ8fCrL8uFyufz65ORkfy6XEzs7O/W6iPAiizOZjH7e09NTdrvdn3EcJ1y8ePHxeDxu37hxIzl79izZtm0bfOrbvHlz84Dj4+ORZDIZgKdQgKEMIHmer9VqNVGSJBOSgeOwNBM9c6EirhFGqI5yA38mk0kdesOGDbrKVqsV1zen4IkTJyKU0gAGrtWuLaWobQCBSvAbMhgA8/Pz+sTwH14IffAMCjZgUKihLl4Ix+Lmwrdjx47mFTx69GikVCoF4C9AQj0sc0gYrA6AgmIABBAK9KLC+j00LIPoAzVhE6ws8CDG4DjOt2fPnuYBDx06FJmamgq43W59YEMtQOIaagEIAHgBhNXIZsBBJdREgOM+FIV60WhUB4cH9+3b1zzgkSNHIolEItDd3a0DQUkDEgrh2ljmAGSoZpxDZQDCk3gBhBnnFy5c0DcPZrPZFwwGmwc8duxYZGZmJgAPITzYJMBfmAwhhZIANnYsBqBRtOE5I3GMREHGI8TwMkK8e/fu5gGPHz+u10G8LSbD4G63e14URZUxxjC5AYVzff1c3F7hE81QvFQqSdFolFu5ciVJJBJk9erVCLtvYGCgecCRkZHI6OioniRQC4Ber/c1l8v1RWtraxmZDH/9VTOyGd/hfHR09JPp6el+URS5sbExsn37dt2DN5XFqIOxWCyAcgKlurq65h0Ox/M8z3/h9/uvNrorQb+TJ09+mEqlnhFFkUeI+/r6kGA3V6hjsVh/vV5/S1XV1chWjuOOUkrf7+npmV4OHPomk8n7K5XKxwsLCy1IHEEQwm1tbRGv11tseqlbLsSt7r/kZuFWT7jc8f41wHA4/DDRtPt4UQwPDQ2lbrsQh0OhMcbYXSZKh14JBg/fdoChUOhnwlgfoTQYDAbD/wMu1/xNK/hBKNRdo/Q+qmlLJpBGac3pdH49ODh47V+jv2kfhUL2WUofopp27acfNhWU7meMuQmln5sY+9a4Tym9MhwMfnP9+o9jhg8e/JwR8mSjilCT6YW9e/d+cKP+74VC+zXGXm10TIlS34uLO5w/qfTewYODjNLhxd++NxyTUlozcdzQ8PDw2I06hkKhBwkhbxDGritICNlICDETSqcJY9f/p6GEXOmm9NGBYHAOYy4Zxkbfern9mvbgcidqtv//gM0qZzz3X1DwI0LILkLIzmAweOK2W0kajcC/lsWNAv4Gk1h1Vt76r0IAAAAASUVORK5CYII="
+          || val.substr(0, BlankImagePrefix.length) === BlankImagePrefix
+          || val.substr(0, FlowImagePrefix.length) === FlowImagePrefix
```

Comment:
Could also use [startsWith](https://caniuse.com/?search=startsWith).

Reply:
This code was removed entirely and replaced with a migration.